Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
Thanks for jumping @cawel -- not 100% sure I understand those points, but happy to discuss on another channel rather than extending this long thread.